File: Day_1.py

numbers = []
f = open("1-1_input.txt", "r")
for line in f:
    line = int(line.strip())
    if numbers:  # Empty list == false
        pass
    else:
        pass
    numbers.append(line)
for i in range(len(numbers)):
    for j in range(len(numbers)):
        if i != j:
            if numbers[i] + numbers[j] == 2020:
                n1 = numbers[i]
                n2 = numbers[j]
                print("Match found! %s + %s = 2020, %s * %s = %s" %(n1, n2, n1, n2, n1 * n2))
        for k in range(len(numbers)):
            if i != j and k != i and k !=j:
                if numbers[i] + numbers[j] + numbers[k] == 2020:
                    n1 = numbers[i]
                    n2 = numbers[j]
                    n3 = numbers[k]
                    print("Match found! %s + %s + %s = 2020, %s * %s * %s = %s" %(n1, n2, n3, n1, n2, n3, n1 * n2 * n3))

File: Day_1.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main () {
    ifstream f("inputs\\Day_1_input.txt");
    f.open("inputs\\Day_1_input.txt", ios::in);
    char data[100];
    cout << "Reading" << endl;
    f >> data;
    cout << data << endl;
    //for (int index = 0; index < 5; ++index)
    //  cout << line << endl;
    f.close();
    
    return 0;
}